polyfactory/factories/base.py

```diff
@@ -672,7 +703,7 @@ def process_kwargs(cls, **kwargs: Any) -> dict[str, Any]:
         """
         result: dict[str, Any] = {**kwargs}
         generate_post: dict[str, PostGenerated] = {}
-
+        cls._check_declared_fields_exist_in_model()
```

Comment:
This should probably move into the `__init_subclass__` so that this check is done when the factory is defined. Also, it should be placed in an appropriate position within there after the rest of the setup and only if it's not a base factory. Keeping it here will cause this check to run repeatedly when calling `Factory.build()`.

Reply:
will look into that on Monday 👌🏻

Reply:
@guacs should be better now :wink: